src/jit/flowgraph.cpp

```diff
@@ -13490,13 +13490,15 @@ bool Compiler::fgOptimizeSwitchBranches(BasicBlock* block)
         GenTree* switchVal = switchTree->gtOp.gtOp1;
         noway_assert(genActualTypeIsIntOrI(switchVal->TypeGet()));
 
+#ifndef LEGACY_BACKEND
```

Comment:
You mean add a comment here or in the code? That operator doesn't exist with LEGACY and all other places where it's referenced are already wrapped in the #ifndef so these changes simply make the definition in gtlist.h and the reference in flowgraph.cpp be wrapped like all the other already are. Does that make sense?